src/App/Fossa/Init/fossa-deps.yml

```diff
@@ -0,0 +1,42 @@
+# # fossa-deps file for fossa-cli
+# # ------------------------------
+# # With the fossa-deps file, you can **manually** include
+# # dependencies, stub dependencies that cannot be **automatically**
+# # discovered with the `fossa analyze` command.
+# #
+# # Dependencies denoted in this file will be automatically
+# # included with dependencies discovered from the `fossa analyze` command.
+# # Any dependencies originating from `fossa-deps` will be shown as a direct
+# # dependency in FOSSA.
+# #
+# # To learn more:
+# # - https://github.com/fossas/fossa-cli/blob/master/docs/references/files/fossa-deps.md
+# #
+# # If you need assistance or would like to file a defect, contact support at:
+# # - https://support.fossa.com/
+# #
+# # -----
+#
+#
+# referenced-dependencies:
```

Comment:
Same comment here: each entry should have a short explanation of what it does and a link to the docs if it's too much to explain in a short comment.